ironic/drivers/modules/drac/management.py

```diff
@@ -389,3 +402,67 @@ def known_good_state(self, task):
         client = drac_common.get_drac_client(node)
         client.reset_idrac(force=True, wait=True)
         client.delete_jobs(job_ids=[_CLEAR_JOB_IDS])
+
+    @METRICS.timer('DracManagement.clear_job_queue')
+    @base.clean_step(priority=0)
+    def clear_job_queue(self, task):
+        """Clear the job queue.
+         :param task: a TaskManager instance containing the node to act on.
+        :returns: None if it is completed.
+        :raises: DracOperationError on an error from python-dracclient.
+        """
+        node = task.node
+
+        client = drac_common.get_drac_client(node)
+        client.delete_jobs(job_ids=[_CLEAR_JOB_IDS])
+
+    @METRICS.timer('DracRAID._query_management_config_job_status')
+    @periodics.periodic(
+        spacing=CONF.drac.query_management_config_job_status_interval)
+    def _query_management_config_job_status(self, manager, context):
+        """Periodic task to check the progress of MANAGEMENT config jobs."""
+
+        filters = {'reserved': False, 'maintenance': False}
+        fields = ['driver_internal_info']
+
+        node_list = manager.iter_nodes(fields=fields, filters=filters)
+        for (node_uuid, driver, driver_internal_info) in node_list:
```

Comment:
If this change approves by chris then, you need to include conductor_group as the third tuple, according to latest master change.

Please refer link : [#L954](https://github.com/openstack/ironic/blob/master/ironic/drivers/modules/drac/raid.py#L954)

Reply:
> If this change approves by chris then, you need to include conductor_group as the third tuple, according to latest master change.
> 
> Please refer link : [#L954](https://github.com/openstack/ironic/blob/master/ironic/drivers/modules/drac/raid.py#L954)

Okay, thanks